### PR TITLE
Clean code and fix indentation

### PR DESCRIPTION
**Pull request by daniv-msft**

### CODE REVIEW

File: Gateway/Startup.cs [hunk 1]
```diff
@@ -3,15 +3,13 @@
 // --------------------------------------------------------------------------------------------
```
Comment (daniv-msft) [inline Gateway/Startup.cs:3]: FYI, I sent a request to our CELA team about these headers (do we need them for open source code?). I will either remove them all or add them in all files in a future PR depending of their reply. #WontFix

Comment (daniv-msft) [reply Gateway/Startup.cs:3]: I received a reply from CELA: we need these headers, and there might actually be other things we have to do... I'll track this in another PR.

---
In reply to: [272772514](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/30#discussion_r272772514) [](ancestors = 272772514)

**Review by ynambiar — APPROVED** [review]
Looks good to me!

**Comment by daniv-msft** [comment]
Thanks Yamini! Checking-in the code.

**Pull request merged**